cupy/__init__.py

```diff
@@ -275,7 +275,6 @@ def is_available():
 from cupy._manipulation.tiling import tile  # NOQA
 
 from cupy._manipulation.add_remove import append  # NOQA
-from cupy._manipulation.add_remove import delete
```

Comment:
Could you add this back here, with `# NOQA` comment? This is needed to make the API public as `cupy.delete`.

Reply:
Also add `delete` under `Adding and removing elements` section of the docs (`docs/source/reference/manipulation.rst`):
https://docs.cupy.dev/en/stable/reference/manipulation.html#adding-and-removing-elements

c.f. https://numpy.org/doc/stable/reference/routines.array-manipulation.html